schemas/telemetry/new-profile/new-profile.4.schema.json

```diff
@@ -875,6 +893,12 @@
                 }
               },
               "type": "object"
+            },
+            "virtualMaxMB": {
```

Comment:
This field does need to allow nulls, at least based on the `validation/telemetry/new-profile.4.sample.pass.json` example.